virt/virt-2-4-release-notes.adoc

```diff
@@ -122,6 +122,28 @@ single sidebar menu item labeled *Virtualization*. When you click *Virtualizatio
 
 [id="virt-2-4-known-issues"]
 == Known issues
+* When upgrading an {product-title} cluster from version 4.4 to 4.5, and container-native virtualization 2.3 is installed, a migrating VMI fails when the target node is evicted due to the upgrade.
+This is because the virt-launcher Pod does not successfully notify the virt-handler Pod that migration has failed.
+The result is that the source VMI `migrationState` is not updated, and the source node remains in a `Ready,SchedulingDisabled` state.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1859661[*BZ#1859661*])
++
+** As a workaround, delete the virt-handler Pod on the source node where the VMI is running. This restarts the virt-handler Pod, which updates the VMI status and migration will be re-triggered:
```

Comment:
s/migration will be re-triggered/restarts migration

maybe? if not, then maybe "triggers migration again"